core/src/main/java/com/readme/android/core/util/BindingAdapters.kt

```diff
@@ -0,0 +1,43 @@
+package com.readme.android.core.util
+
+import android.net.Uri
+import android.widget.EditText
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.core.content.ContextCompat
+import androidx.databinding.BindingAdapter
+import com.readme.android.core.R
+import com.readme.android.core.constant.*
+
+object BindingAdapters {
+
+    @JvmStatic
+    @BindingAdapter("nickNameStateNumber")
+    fun setNickNameState(textView: TextView, nickNameStateNumber: Int?) {
+        val context = textView.context
```

Comment:
바인딩 어댑터의 경우 확장함수 형식으로
```suggestion
    fun TextView.setNickNameState(nickNameStateNumber: Int?) {
        val context = this.context
```
`textView`라는 파라미터를 계속 함수내에서 호출할 필요없이 생략가넝~!

Reply:
이거 알고는 있었는데 습관적으로 이렇게 쓰는데 고쳐볼까?

Reply:
뭐 고치면.. 밑에 textView.setText() 이런것들도 setText() 이렇게만 호출해도 되겄지..!